garg2021re/journal/metadata.yaml

```diff
@@ -1,139 +1,123 @@
-
-# Scientific domain of the article (e.g. Computational Neuroscience)
-#  (one domain only & try to be not overly specific)
-domain: 
-
-# Coding language (main one only if several)
-language:
-
-  
-# To be filled by the author(s) after acceptance
-# -----------------------------------------------------------------------------
-
-# For example, the URL of the GitHub issue where review actually occured
-review: 
-  - url: 
-
-contributors:
-  - name:
-    orcid: 
-    role: editor
-  - name:
-    orcid:
-    role: reviewer
-  - name:
-    orcid:
-    role: reviewer
-
-# This information will be provided by the editor
-dates:
-  - received:  November 1, 2018
-  - accepted:
-  - published: 
-
-# This information will be provided by the editor
-article:
-  - number: # Article number will be automatically assigned during publication
-  - doi:    # DOI from Zenodo
-  - url:    # Final PDF URL (Zenodo or rescience website?)
-
-# This information will be provided by the editor
-journal:
-  - name:   "ReScience C"
-  - issn:   2430-3658
-  - volume: 4
-  - issue:  1
+# To be filled by the author(s) at the time of submission
+# -------------------------------------------------------
+
+# Title of the article:
+#  - For a successful replication, it should be prefixed with "[Re]"
+#  - For a failed replication, it should be prefixed with "[¬Re]"
+#  - For other article types, no instruction (but please, not too long)
+title: "[Re] Training Binary Neural Networks using the Bayesian Learning Rule"
+
+# List of authors with name, orcid number, email and affiliation
+# Affiliation "*" means contact author (required even for single-authored papers)
+authors:
+  - name: Prateek Garg
+    orcid: 0000-0003-3162-4011
+    email: prateekgarg.iitd@gmail.com
+    affiliations: 1,*
+    
+  - name: Lakshya Singhal
+    orcid: 0000-0002-3461-2529
+    email: lakshyasinghaliitd@gmail.com
+    affiliations: 1      # * is for contact author
+
+  - name: Ashish Sardana
+    orcid: 
+    email: asardana@nvidia.com
+    affiliations: 2      # * is for contact author
+
+
+
+# List of affiliations with code (corresponding to author affiliations), name
+# and address. You can also use these affiliations to add text such as "Equal
+# contributions" as name (with no address).
+affiliations:
+  - code:    1
+    name:    Indian Institute of Technology Delhi
+    address: New Delhi, India
+    
+  - code:    2
+    name:    NVIDIA
+    address: Bangalore, KA, India
+    
+
+# List of keywords (adding the programming language might be a good idea)
+keywords:  rescience c, machine learning, deep learning, binary neural network, python, pytorch
+
+# Code URL and DOI/SWH (url is mandatory for replication, doi after acceptance)
+# You can get a DOI for your code from Zenodo, or an SWH identifier from
+# Software Heritage.
+#   see https://guides.github.com/activities/citable-code/
+code:
+  - url: https://github.com/prateekstark/training-binary-neural-network
+  - doi: 
+  - swh: 

```

Comment:
Please add the Software Heritage Foundation (SWH) uri as described in the readme.